user_management/api/tests/test_serializers.py

```diff
@@ -161,7 +162,8 @@ def test_deserialize_mismatched_passwords(self):
         self.data['password2'] = 'different_password'
         serializer = serializers.RegistrationSerializer(data=self.data)
         self.assertFalse(serializer.is_valid())
-        self.assertIn('password2', serializer.errors)
+        message = 'Your passwords do not match.'
+        self.assertIn(message, serializer.errors['non_field_errors'])
```

Comment:
What is `serializer.errors['non_field_errors']` here? If it's `'Your passwords do not match.'`, this should be `self.assertEqual`.